test/iam.test.js

```diff
@@ -0,0 +1,47 @@
+const { MongoClient } = require('mongodb');
+const mockSES = require("aws-sdk/clients/ses");
+const iam = require("../commands/iam");
+
```

Comment:
this is probably the best stuff in the PR, mocking SES and mongo is really pro level testing and will help a lot for development